### PR TITLE
Read time calculation

### PR DESCRIPTION
**Pull request by sean-binary**
Changes:

-   dynamically calculate read time based on blog post content

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/academy/_recent-featured-posts.js [hunk 1]
```diff
@@ -81,8 +81,9 @@ const RecentFeaturedPosts = ({ recent_data, featured_data }) => {
                                         {headline_recent?.published_date &&
                                             convertDate(headline_recent?.published_date)}
                                         <ClockIcon src={Clock} />
-                                        {headline_recent.read_time_in_minutes}{' '}
-                                        <Localize translate_text="min read" />
+                                        {calculateReadTime(stripHTML(headline_recent?.blog_post)) +
+                                            ' ' +
+                                            localize('min read')}
```
Comment (njazuli-deriv) [inline src/pages/academy/_recent-featured-posts.js:86]: Can we directly return this string from `calculateReadTime` function?

Comment (sean-binary) [reply src/pages/academy/_recent-featured-posts.js:86]: Yep, that'll be better. I will shift `stripHTML` in there as well since they are all using it.

**Review by prince-deriv — APPROVED** [review]

File: src/common/utility.js [hunk 2]
```diff
@@ -325,6 +325,14 @@ export const replaceLocale = (locale) => {
     return checked_locale
 }
 
+export const stripHTML = (str) => str?.replace(/<[^>]*>?/gm, '')
+
+export const calculateReadTime = (text) => {
+    const wpm = 275 // average adult words per minute
```
[the remark this answers is not in the document]
Comment (sean-binary) [reply src/common/utility.js:332]: This was initially from an online source, but was later tweaked as the min count was a little too large for the content team's liking. Shall I remove the comment then?

**Review by prince-deriv — APPROVED** [review]
LGTM

**Pull request merged**